### DISCUSSION TITLE
libnpmpublish

### DISCUSSION THREAD

**Discussion by ayuhito** [DefinitelyTyped/DefinitelyTyped#54758] [Request a new @types package] [answered]
https://www.npmjs.com/package/libnpmpublish

**Comment by nicolas377** [comment, marked as answer]
I'll start working on that after my current pr (fixes a small issue in @types/chrome, #54823) gets merged!

**Reply by nicolas377** [reply]
#54862 is adding that. if you (@DecliningLotus) want, I can add you as a owner!

**Reply by ayuhito** [reply]
> #54862 is adding that. if you (@DecliningLotus) want, I can add you as a owner!

There's no reason to 😄 You did all the work. Thank you so much!

Edit: I'll give it a review later today. I've never contributed to DefinitelyTyped before.
